OpenOversight/app/main/views.py

```diff
@@ -263,9 +263,14 @@ def redirect_sort_images(department_id: int):
 )
 @login_required
 def sort_images(department_id: int):
+    try:
+        department = Department.query.filter_by(id=department_id).one()
+    except NoResultFound:
+        abort(HTTPStatus.NOT_FOUND)
```

Comment:
Though it is equivalent logic, I feel more comfortable using the `id` field of the `Department` object to query the `Image`s downstream.

Reply:
True, one more additional query. I think there's more value to the sureness around it than the minor performance hit.